.env

```
@@ -1,2 +1,3 @@
 FUSION_SERVER_URL=http://localhost:8000
+AUDITS_SERVER_URL=http://localhost:8080
```

`[Issue]` Should point to the Gateway, not the audits service.


```suggestion
GATEWAY_URL=http://localhost:8000
```